go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/kubeflow/pipelines
 
 require (
+	cloud.google.com/go v0.89.0
```

Comment:
Rethinking over this, there are people importing KFP module in go.
Therefore, it's probably a bad idea to add another dependency just because we need it in our tools.
I'd suggest creating a separate `go.mod` file in the project cleaner folder, so that it can be a self contained go module.

Reply:
I agree, created separate go.mod for project cleaner folder.